back/engines/commercial/analytics/app/controllers/analytics/web_api/v1/analytics_controller.rb

```diff
@@ -5,7 +5,8 @@
 module Analytics
   module WebApi::V1
     class AnalyticsController < ::ApplicationController
-      def create
+      skip_after_action :verify_policy_scoped, only: :index
```

Comment:
@adessy this is probably wrong, how should i used scopes here? without this im getting a `Pundit::PolicyScopingNotPerformedError` error

Reply:
To make it work with index, you do need to skip the `:verify_policy_scoped check`, just like you did. But ideally, you should replace it with the same verification hook used by `create` and the other actions by adding :
```ruby
after_action :verify_authorized, only: :index
```